modules/custom/openy_repeat_entity/src/Entity/RepeatInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Drupal\openy_repeat_entity\Entity;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+
+/**
+ * Provides an interface defining a Repeat entity.
+ * @ingroup openy_repeat_entity
+ */
+interface RepeatInterface extends ContentEntityInterface {
+
+}
```

Comment:
please make all files 644, not 755
modules/custom/openy_schedules/js/openy_schedules.js